kratos/processes/apply_ray_casting_process.h

```diff
@@ -320,6 +323,10 @@ class KRATOS_API(KRATOS_CORE) ApplyRayCastingProcess : public Process
     ///@name Un accessible methods
     ///@{
 
+private:
+    KRATOS_REGISTRY_ADD_PROTOTYPE("Processes.KratosMultiphysics", ApplyRayCastingProcess<TDim>)
+    KRATOS_REGISTRY_ADD_PROTOTYPE("Processes.All", ApplyRayCastingProcess<TDim>)
+
```

Comment:
Though these can be perfectly placed in here, I'd put them together to the pointer macro. I've the feeling that this would be simpler for the newbies.

Reply:
ok , done!